### PR TITLE
Add profile 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, the design part of a website is not important enough.
- Currently, customers can not log in to see their analysis.
- Currently, the Result Chart (Signet) does not look like the website and is unfit for use in physical analysis.
- Currently, the greeting is only fitting for existing members. (Welcome back)
- Currently, there is much content in the ProfilePage, which does not really apply to our Atomic Design philosophy.
- Currently, the footer contains some useless links and features.
- Currently, the Result Chart (Signet) is being initialized in the ReactJS `render()` function, which sets state within the `render()` function. This is against the way.
- Currently, the password set for customers is lowercase, which can be confusing when printing.

**What is the new behavior (if this is a feature change)?**
- The weight of the design part of a website has been increased.
- Customers can now log in to see their analysis.
- Customers can now see details regarding their analysis.
- The design of the Result Chart (Signet) has been improved to fit the website.
- The greeting has been improved to fit all new users as well. (Welcome)
- The ProfilePage has been split up in multiple lower-level components to please the Atomic Design gods.
- The footer now only contains copyright information.
- The Result Chart (Signet) is now being initialized in the `componentDidMount = () => {}` function. This is the way.
- The password set for customers is now being converted to uppercase. This is the way.

**Other information**:
- The version has been updated to v1.3.0.

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks good. But I could... be... preoccupied.

File: src/store/reducers/authReducer.js [hunk 1]
```diff
@@ -43,6 +44,7 @@ const authReducer = (state = initState, action) => {
         upgradeFailed: false,
       };
     case "UPGRADE_ERROR":
+      console.error("Could not upgrade", action.err);
```
Comment (schettn) [inline src/store/reducers/authReducer.js:48]: add newline or remove console.log

File: src/store/reducers/authReducer.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ const authReducer = (state = initState, action) => {
         zombies: action.zombies,
       };
     case "GETZOMBIE_ERROR":
+      console.error("Could not get zombies", action.err);
```
Comment (schettn) [inline src/store/reducers/authReducer.js:36]: add newline or remove console.log

**Review by schettn — CHANGES_REQUESTED** [review]
almost-but-not-quite

**Review by LucaAllmaier — APPROVED** [review]
Looks good.

File: src/components/molecules/Services/index.jsx [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+  },
+  { name: "Mein Image", color: "green", icon: "award" },
+  { name: "Meine Präsenzen", color: "deep-purple", icon: "bullhorn" },
+];
+
+class Services extends React.Component {
+  state = { activeItem: 0 };
+
+  componentDidMount = () => {
+    //this.getAnalytics();
```
Comment (pinterid) [inline src/components/molecules/Services/index.jsx:49]: Remove this comment or add a newline after it.

File: src/components/organisms/ZombieList/zombie.scss [hunk 4]
```diff
@@ -2,8 +2,15 @@
...
+  .badge {
+    box-shadow: none;
+  }
+}
+.modal-zombie {
+  .progress-bar {
+    background-color: #f61a42 !important;
+  }
 }
-
```
Comment (pinterid) [inline src/components/organisms/ZombieList/zombie.scss:6]: Missing new line.

**Review by pinterid — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by sweggerat — APPROVED** [review]
looks good

**Review by schettn — APPROVED** [review]
LGTM!

**Review by pinterid — APPROVED** [review]
Looks good to merge.

File: src/components/molecules/Services/index.jsx [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
+              {profile.us && (
+                <>
+                  {profile.us.web ? (
+                    <MDBAlert color="info">
+                      Deine Website wird von uns gewartet.
+                    </MDBAlert>
+                  ) : (
+                    <MDBAlert color="warning">
+                      Deine Website wurde nicht von uns erstellt.
+                    </MDBAlert>
```
Comment (kleberbaum) [inline src/components/molecules/Services/index.jsx:210]: warning: Deine Website wurde nicht von uns erstellt. xD

**Review by kleberbaum — COMMENTED** [review]
lmao xD

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**